freud/density.pyx

```diff
@@ -593,27 +600,23 @@ cdef class RDF(_SpatialHistogram1D):
         r_min (float, optional):
             Minimum interparticle distance to include in the calculation
             (Default value = :code:`0`).
-        normalize (bool, optional):
-            Scale the RDF values by
-            :math:`\frac{N_{query\_points}}{N_{query\_points}-1}`. This
-            argument primarily exists to deal with standard RDF calculations
-            where no special ``query_points`` or ``neighbors`` are provided,
-            but where the number of ``query_points`` is small enough that the
-            long-ranged limit of :math:`g(r)` deviates significantly from
-            :math:`1`. It should not be used if :code:`query_points` is
-            provided as a different set of points, or if unusual query
-            arguments are provided to :meth:`~.compute`, specifically if
-            :code:`exclude_ii` is set to :code:`False`. This normalization is
-            not meaningful in such cases and will simply convolute the data.
-
+        normalization_mode (str, optional):
+            There are two valid string inputs for this argument. The first
+            option, ``infer``, handles the normalization as shown mathematically
+            at the beginning of this class's docstring. The other option,
+            ``finite_size``, adds an extra rescaling factor of
+            :math:`\frac{N_{query\_points}}{N_{query\_ponts} - 1}` so the RDF
+            values will tend to 1 at large :math:`r` for small systems (Default
+            value = :code:`'infer'`).
```

Comment:
We actually need to do a little more work for `infer`. The current behavior of `infer` is what we want if `query_points is points` (or equivalently if the user doesn't specify `query_points`). In the case where `query_points != points`, though, I believe that we should instead be normalizing by `N_query_points * N_points`.

Reply:
Is your comment here suggesting that we implement the new normalizaiton solution in this PR? You've previously stated (see your last comment in #635) that we may not want to do that in order to move things along a bit quicker.